recipes/r-clusteval/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = '0.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-clusteval
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/clusteval_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/clusteval/clusteval_{{ version }}.tar.gz
+  sha256: 329f0f1fd5b9fcd1fa253d0c8d1373d7b0eed919eacc51bfd30d7947685ba0d1
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}make
+    - {{posix}}sed               # [win]
+    - {{posix}}coreutils         # [win]
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-rcpp >=0.9.13
+    - r-mvtnorm
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
+    - r-rcpp >=0.9.13
+    - r-mvtnorm
+
+test:
+  commands:
+    - $R -e "library('clusteval')"           # [not win]
+    - "\"%R%\" -e \"library('clusteval')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=clusteval
+  license: MIT
+  license_file: LICENSE
+  summary: An R package that provides a suite of tools to evaluate clustering algorithms, clusterings,
+    and individual clusters.
+  license_family: MIT
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - khughitt
```

Comment:
Please do not remove the commented cran metadata that is normally added here at the end. It helps reviewing the recipe.